sidebars.js

```diff
@@ -116,6 +116,15 @@ const sidebars = {
       "collapsed": true,
       "label": 'Troubleshooting',
       "items": [
+        {
+          "type": "category",
```

Comment:
Could we make this not a category? I dislike having these single-item collapsible with only one sub-item.. Would much rather have all the `Troubleshooting` sub-items be links :+1: